tests/db/test_v202208190_to_v202209160.py

```diff
@@ -0,0 +1,42 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import os
+from typing import Awaitable, Callable, List
```

Comment:
We concluded that for new code we would use `from collections import abc` and then use `abc.Awaitable`, `abc.Callable`, `abc.Sequence`, ... where applicable and `list`, `dict` where required.